semgrep/semgrep/test.py

```diff
@@ -169,35 +169,30 @@ def confusion_matrix_to_string(confusion: List[int]) -> str:
 
 
 def invoke_semgrep(
-    verbose: bool, strict: bool, test_files: List[Path], config: Path, unsafe: bool
+    strict: bool, test_files: List[Path], config: Path, unsafe: bool
 ) -> Any:
     return json.loads(
         semgrepmain(
-            argparse.Namespace(
-                verbose=verbose,
-                strict=strict,
-                dump_ast=False,
-                no_rewrite_rule_ids=True,
-                dangerously_allow_arbitrary_code_execution_from_rules=unsafe,
-                config=str(config),
-                quiet=True,
-                precommit=False,
-                generate_config=False,
-                pattern=None,
-                validate=False,
-                json=True,
-                sarif=False,
-                exclude_tests=False,
-                jobs=1,
-                exclude=[],
-                include=[],
-                exclude_dir=[],
-                include_dir=[],
-                output=None,
-                error=False,
-                autofix=False,
-                target=[str(t) for t in test_files],
-            )
+            target=[str(t) for t in test_files],
```

Comment:
Is there any way we can just grab the `argparse` defaults here? Currently all these values are duplicated between `semgrep.cli.cli` and here. I always forget to update this spot when adding a new argument and am reminded to duplicate it here.

Reply:
With this PR you actually get alerted on missing args with mypy.

Reply:
Probably going to switch over to using click in the near future so that should remove the duplication so fine with mypy checked duplication for now.